src/components/CompareResults/RunValues.tsx

```diff
@@ -87,7 +86,9 @@ function RunValues(props: RunValuesProps) {
         {measurementUnit}
       </div>
       <div className={styles.deviation}>
-        {stddev} {unit} = {stddevPercent}% standard deviation
+        {values.length > 1
+          ? `${stddev} ${unit} = ${stddevPercent}% standard deviation`
+          : `${stddev} ${unit} = N/A standard deviation`}
```

Comment:
Currently we still print `${stddev} ${unit}` but I think we should remove it completely and just keep `N/A standard deviation`. Because `stddev` is the standard deviation value, and seeing it as `0 MS = N/A` might still confuse some people. Zero here means that we failed to calculate it.

Reply:
@gmierz This looks good to me, after the PR is rebased in my opinion we could merge it. Do you have other feedback?

Reply:
This looks great to me, thanks! :)